app/web/features/profile/view/Badges.tsx

```diff
@@ -0,0 +1,46 @@
+import { Chip, Tooltip } from "@material-ui/core";
+import { User } from "proto/api_pb";
+import makeStyles from "utils/makeStyles";
+
+import { useBadges } from "../hooks/useBadges";
+
+interface Props {
+  user: User.AsObject;
+}
+
+const useStyles = makeStyles((theme) => ({
+  badgeContainer: {
+    marginTop: theme.spacing(1),
+  },
+  badge: {
+    marginInlineStart: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+export const Badges = ({ user }: Props) => {
+  const classes = useStyles();
+  const { badges } = useBadges();
+  console.log(user);
```

Comment:
console.log trace 👀

Reply:
Oop, gotta fix that. I was having issues with the protos not updating somehow and with the docker containers, so I resorted to stone age debugging :P